pkg/workloads/cortex/serve/init/bootloader.sh

```diff
@@ -46,22 +46,17 @@ if [ -f "/mnt/project/.env" ]; then
     set +a
```

Comment:
do you think we should also `rm -rf /mnt/workspace/proc-*-ready.txt` after the `rm -rf /mnt/workspace/api_readiness.txt` line above?